pages/common/xml-select.md

```diff
@@ -0,0 +1,28 @@
+# xml-select
+
+> Select from XML documents using XPATH.
+> More information: <http://xmlstar.sourceforge.net/doc/xmlstarlet.pdft>.
+
+- TBD:
+
+`xml select`
+
+- TBD:
+
+`xml select`
+
+- TBD:
+
+`xml select`
+
+- TBD:
+
+`xml select`
+
+- TBD:
+
+`xml select`
```

Comment:
Again, this looks like a placeholder. As before, I recommend either:

1. Filling these out
2. Removing this page from this PR and potentially contributing it in a completed state in a future PR

Reply:
Please read the PR comment.

Reply:
This has been resolved.